mk/docs.mk

```diff
@@ -288,3 +292,9 @@ doc/style/index.html: $(RUSTBOOK_EXE) $(wildcard $(S)/src/doc/style/*.md) | doc/
 	@$(call E, rustbook: $@)
 	$(Q)rm -rf doc/style
 	$(Q)$(RUSTBOOK) build $(S)src/doc/style doc/style
+
+error-index: doc/error-index.html
```

Comment:
This rule doesn't seem to be included from anywhere, perhaps it could be added to the `docs` rule so this is prepared automatically with the rest of the documentation?